### ISSUE TITLE
The latest release version (0.16.2) has almost no important buttons working at launcher

### ISSUE THREAD

**Issue by ReginaldCZ** [Interkarma/daggerfall-unity#2549]
**Describe the issue**

Many important buttons aren't working at the launcher GUI.:

1. The close button at advanced settings, which makes me unable to change any advanced settings without having to edit them manually in their text file.
2. The "OK" button when confirming the installation location of Daggerfall, making me unable to even progress next.
3. The "Play" button, making me unable to even launch the game, making the game unplayable

**To Reproduce**

Basic installation.:
1. Download the Daggerfall Unity release, Windows version 0.16.2. (at this time the latest release version).
2. Try setting up the new installation of Daggerfall.
3. Confirm it.
4. Notice that confirm button does nothing.

Attempting to close advanced settings menu.:
1. Try changing the advanced settings of the game through the GUI.
2. Try now closing the menu.
3. Notice that the button is broken.

Attemting to launch the game.:
1. Try just launch the game through the GUI.
2. Nope, even the play button doesn't work.

**Expected behavior**

The buttons working as supposed.

**Desktop (please complete the following information):**

 - OS:Windows
 - Version [e.g. 0.16.2]

**Additional context**

When trying to do same stuff at the previous release version (0.16.1.), there the buttons work as supposed.

**Comment by Interkarma** [comment]
Hey there. :) This is not observed in the current release, and will be related to a setup/mod issue. Please use the forums for individual support:

https://forums.dfworkshop.net/index.php

**Comment by carlos-montano-hub** [comment]
> The "OK" button when confirming the installation location of Daggerfall, making me unable to even progress next.

@Interkarma Having this exact issue as well

I am using the steam version and have a 1080p monitor alongside a 1440p monitor on W10

**Comment by carlos-montano-hub** [comment]
@ReginaldCZ And for reference deleting the [appdata folder](https://github.com/Interkarma/daggerfall-unity/issues/2546#issuecomment-1821528069_)  helped solving this

**Comment by Interkarma** [comment]
> > The "OK" button when confirming the installation location of Daggerfall, making me unable to even progress next.
> 
> @Interkarma Having this exact issue as well
> 
> I am using the steam version and have a 1080p monitor alongside a 1440p monitor on W10

I would welcome either of you to post to Help & Support the forums, that's the correct place for individual support. Including a zip of your player.log and any screenshots would also helpful. :)

https://forums.dfworkshop.net/

The issues area here is for the development team to track bugs that are known and reproduceable. At time of writing nobody has been able to reproduce the problem you're experiencing. Until then, the forums are the correct place for it. I promise that I'll do everything I can to work with you to discover what's going on.

**Comment by Interkarma** [comment]
> @ReginaldCZ And for reference deleting the [appdata folder](https://github.com/Interkarma/daggerfall-unity/issues/2546#issuecomment-1821528069_) helped solving this

Is there any chance you could recover the contents of AppData for me to try and reproduce problem?

Deleting _settings.ini_ is a common cure-all for all manner of setup problems, so it's possible just recovering this file might be enough to trigger issue. If I can reproduce, I can try to mitigate it.

**Comment by Interkarma** [comment]
My suspicion is this will be related to deploying/loading settings.ini. If there's some edge case where DFU proceeds to open without a valid settings file, that would no doubt cause issues with the launcher. The problem will be if the game continues to start even when settings are not valid.

I've tried several methods of triggering invalid settings at start and haven't been able to recreate the exact issue described here. I did manage to reproduce a different problem where settings exist but were totally empty (which I've now fixed), but it doesn't match the problem described here.

@ReginaldCZ if you are able to recover that settings file it might shed some more light on things. Otherwise, I'll just need to wait for someone else to raise problem and hopefully get a live copy of their settings or AppData. Thanks for your feedback in any case, most helpful. :)
